Atlassian.Jira/Remote/JiraRestClient.cs

```diff
@@ -258,46 +260,41 @@ private JToken GetValidJsonFromResponse(IRestRequest request, IRestResponse resp
             }
         }
 
-        private async Task<string> GetResponseHeadersAsync(IRestRequest request)
+        private async Task<JToken> FallbackToHttpClientOnFailureAsync(IRestRequest request)
         {
-            try
+            using var handler = new HttpClientHandler
             {
-                using var handler = new HttpClientHandler()
-                {
-                    AutomaticDecompression = DecompressionMethods.GZip | DecompressionMethods.Deflate
-                };
-                handler.ServerCertificateCustomValidationCallback += (sender, certificate, chain, errors) => true;

```

Comment:
why did you remove? aren't we ignoring SSL errors?

Reply:
I moved it to the constructor... look at L268 in the new version